build.sbt

```diff
@@ -326,14 +326,15 @@ lazy val notificationworkerlambda = project
   .enablePlugins(RiffRaffArtifact)
   .settings(
     libraryDependencies ++= Seq(
-      "com.typesafe.play" %% "play-logback" % "2.6.16",
+      "com.amazonaws" % "aws-lambda-java-log4j2" % "1.1.0",
```

Comment:
How come you need the aws log4j dependency for just the worker lambda and not the others?

Reply:
I didn't add it if the dependency was already there :)

Reply:
:+1: